UM/Qt/Bindings/ContainerProxy.py

```diff
@@ -1,4 +1,5 @@
-from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
+from PyQt5.QtCore import QObject
+from UM.FlameProfiler import pyqtSlot
```

Comment:
It is a shame that there is no less intrusive way to handle this. Because we will be forgetting this at one point or another.